frontend/src/components/Course/Course.tsx

```diff
@@ -51,50 +50,53 @@ const Course: React.FC<Props> = ({
   windowWidth,
   type,
   imageUrl,
-  isCourseCompleted,
   courseRef,
   clientCourseRef,
-}) => (
-  <CourseContainer
-    container
-    direction="column"
-    ref={pageName === PAGES.coursesList ? courseRef : clientCourseRef}
-  >
-    <AboutCourseContainer type={type}>
-      <ImageWrapper imageUrl={imageUrl} />
-      <CourseTextContainer>
-        <CourseTitle type={type}>{title}</CourseTitle>
-        <CourseDescriptionWrapper type={type}>
-          <CourseDescription type={type}>
-            {shortifyCourseDescription(description, windowWidth, pageName)}
-          </CourseDescription>
-        </CourseDescriptionWrapper>
-        <MobileCourseInfoBox>
+  progress,
+}) => {
+  return (
+    <CourseContainer
+      container
+      direction="column"
+      ref={pageName === PAGES.coursesList ? courseRef : clientCourseRef}
+    >
+      <AboutCourseContainer type={type}>
+        <ImageWrapper imageUrl={imageUrl} />
+        <CourseTextContainer>
+          <CourseTitle type={type}>{title}</CourseTitle>
+          <CourseDescriptionWrapper type={type}>
+            <CourseDescription type={type}>
+              {shortifyCourseDescription(description, windowWidth, pageName)}
+            </CourseDescription>
+          </CourseDescriptionWrapper>
+          <MobileCourseInfoBox>
+            <CourseInfo duration={duration} lessons={lessons} />
+          </MobileCourseInfoBox>
+          {status === COURSE_STATUSES.completed && (
+            <MobileCourseCompleted>
+              Completed <img alt="" src={checkIcon} />
+            </MobileCourseCompleted>
+          )}
+        </CourseTextContainer>
+        {pageName === PAGES.myCourses && status !== COURSE_STATUSES.completed ? (
+          <MobileCourseProgress>
+            <ProgressBar
+              size={SIZE.medium}
+              text={`${progress}%`}
+              value={progress}
+              textColor={PROGRESS_COLOR}
```

Comment:
It is a constant for text: TEXT_COLOR
PROGRESS_COLOR is for the path and has a green color